lib/client/kadira.js

```diff
@@ -6,17 +6,49 @@ Kadira.disableErrorTracking = function () {
   Kadira.options.enableErrorTracking = false;
 };
 
-Kadira.trackError = function (type, message, { stacks = '' } = {}) {
-  if(Kadira.options.enableErrorTracking && type && message) {
+Kadira.trackError = function () {
+  if (!Kadira.options.enableErrorTracking) {
+    return;
+  }
+
+  let subType = null;
+  let message = null;
+  let stack = null;
+
+  if (
+    !(arguments[0] instanceof Error) &&
+    typeof arguments[0] === 'string' &&
+    typeof arguments[1] === 'string'
+  ) {
+    // Old usage
+    // Monti.trackError('type', 'error message', { stacks: 'error stack' });
+    subType = arguments[0] || 'Monti.trackError';
+    message = arguments[1];
+    stack = arguments[2] && arguments[2].stacks || '';
+  } else {
+    // New usage
+    // Monti.trackError(error, options);
+    const error = arguments[0];
+    const options = arguments[1];
+
+    message = error && error.message;
```

Comment:
An error can be any type of value. This should check only get the message property if the error is an object, and otherwise use the error itself as the message.